domains/Contact/ManageContactName/Web/ViewHelpers/ModuleContactNameViewHelper.php

```diff
@@ -4,20 +4,18 @@
 
 use App\Models\Contact;
 use App\Models\User;
-use Illuminate\Support\Facades\DB;
 
 class ModuleContactNameViewHelper
 {
     public static function data(Contact $contact, User $user): array
     {
-        $entry = DB::table('contact_vault_user')
-            ->where('contact_id', $contact->id)
-            ->where('user_id', $user->id)
+        $entry = $user->contacts()
+            ->wherePivot('contact_id', $contact->id)
             ->first();
 
         return [
             'name' => $contact->name,
-            'is_favorite' => $entry ? $entry->is_favorite : false,
+            'is_favorite' => $entry ? $entry->pivot->is_favorite : false,
```

Comment:
I think we should add that to phpstan.neon because we'll probably have a lot of it in the future.